project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_instance.inc

```diff
@@ -353,93 +353,6 @@ function c4m_content_group_field_default_field_instances() {
     ),
   );
 
-  // Exported field_instance: 'node-group-c4m_group_welcome_message'.
-  $field_instances['node-group-c4m_group_welcome_message'] = array(
```

Comment:
@nirgn975 
This field is less than 24 hours old, it should remain....
Check #903